### DISCUSSION TITLE
Rustdesk pro - Client SSL

### DISCUSSION THREAD

**Discussion by rabih777** [rustdesk/rustdesk-server-pro#399] [Q&A]
Hello,
I'm checking if letting the Rust desk Quick Agent use SSL is possible. I did the configuration for NGINX, but the client is still trying to contact ports 21116, 21117 ... etc. Can the client use port 443 only?

**Comment by rustdesk** [comment]
Nginx set up in our doc is for the web console (port 21114), not for 21116/21117. If you want to use 443 only, you have to set up proxy, https://github.com/rustdesk/rustdesk/wiki/Set-up-http-proxy-server
